lib/yaml_db/serialization_helper.rb

```diff
@@ -195,7 +196,19 @@ def self.table_record_count(table)
         ActiveRecord::Base.connection.select_one("SELECT COUNT(*) FROM #{Utils.quote_table(table)}").values.first.to_i
       end
 
-    end
+      # Just return the first column unless the second column is also an id, in
+      # which case it joins them for ordering
+      def self.table_id(table)
```

Comment:
Actually, a second review. I'm concerned this is a misleading name because it overloads the meaning of `id`. Can you please switch it to `sort_key` or something that indicates its real purpose, and rename it correspondingly in `.each_table_page`? It'll probably make sense to change the internal variables here, too.